### ISSUE TITLE
assets: zopieux must fix his shit

### ISSUE THREAD

**Issue by seirl** [prologin/concours-site#65]
> ###### Originally posted 2016-04-02 15:17 by [serialk](https://bitbucket.org/serialk/) on Bitbucket

    17:15 <zopieux> oublie les assets
    17:15 <zopieux> c'est crade
    17:15 <zopieux> je dois fix cette merde

**Comment by zopieux** [comment]
> ###### Originally posted 2017-03-12 21:36 by [Zopieux](https://bitbucket.org/Zopieux/) on Bitbucket

assets: zopieux fixed his shit (fixes #65)

2016-04-02 17:15 <zopieux> je dois fix cette merde


→ <<cset 1b4479f4d618>>

**Comment by seirl** [comment]
everything is still shit.

```
01:17:32 <seirl> zopieux: make assets marche pas
01:17:41 <seirl> https://www.emojione.com/wp-content/uploads/assets/e1-png.zip]
01:17:42 <seirl> https://www.emojione.com/wp-content/uploads/assets/e1-png.zip
01:17:44 <seirl> 404
01:18:03 <zopieux> ENCORE ?
01:19:24 <seirl> https://d1j8pt39hxlh3d.cloudfront.net/emoji/emojione/3.1.1/EmojiOne_3.1.1_32x32_png.zip
01:19:26 <seirl> https://d1j8pt39hxlh3d.cloudfront.net/emoji/emojione/3.1.1/EmojiOne_3.1.1_64x64_png.zip
01:19:29 <seirl> https://d1j8pt39hxlh3d.cloudfront.net/emoji/emojione/3.1.1/EmojiOne_3.1.1_128x128_png.zip                                                                                                                         
01:19:32 <seirl> tu veux lequel ?
01:19:37 <seirl> elles sont toutes super moches
01:19:44 <seirl> (fallait remplir un formulaire à la con pour avoir les liens)
01:21:29 <seirl> PUTAIN ZOPIEUX 32 64 OU 128 BORDEL DE CUL
01:21:52 <seirl> y'a du avoir beaucoup de gens qui ont fait des tests install du site en local dites moi
01:24:55 <seirl> wtf c'est quoi ce code débile
01:25:03 <seirl> zipball = open('/tmp/e1-png.zip', 'rb')
01:25:07 <seirl> le fichier est pas créé avant
01:25:44 <seirl> et cette variable est unused
01:25:45 <seirl> mais wtf
01:26:03 <zopieux> ah
01:26:04 <zopieux> coucou
01:26:10 <seirl> LE CODE QUI DOWNLOAD EST COMMENTÉ
01:26:13 <zopieux> hein
01:26:28 <zopieux> wait
01:26:46 <zopieux> haha
01:26:48 <zopieux> HAHAHAHAHA
01:26:51 <zopieux> :D
01:27:01 <seirl> https://github.com/prologin/site/commit/1b4479f4d61841cb13df6a8c1fee44a6df4548e8#diff-382918edb62fd07edfc7d20ffb3d9955R36
01:27:04 <zopieux> "lemme do that hack real quick"
01:27:10 <seirl> 12 mars
01:27:16 <seirl> personne a installé le site depuis le 12 mars
01:27:23 <zopieux> oui ça montre bien l'étendu des développeurs site
01:27:26 <zopieux> ah merde burn
01:27:44 <zopieux> en pratique ça devrait être jenkins ou buildbot ou whatever qui installe le site
01:27:56 <zopieux> mais faut LE FAIRE
01:28:20 <seirl> tu comptes fix ta merde ce soir ou je vais me coucher ?
01:28:31 <zopieux> mec t'as juste à décommenter les deux lignes
01:28:43 <seirl> VOYONS VOIR.
01:28:57 <seirl> sondage #prologin-staff :
01:29:05 <seirl> qui pense que j'ai « juste à décommenter les deux lignes » ?
01:29:15 <zopieux> la ligne en l'occurrence
01:29:33 <seirl> FileNotFoundError: [Errno 2] No such file or directory: '/tmp/e1-png.zip'
01:29:36 <seirl> wow
01:29:40 <seirl> such decommenting
01:29:56 <zopieux> ah non mais jouer au con c'est pas du jeu
01:30:01 <zopieux> ton sondage est biaisé
01:30:12 <seirl> t'as plein d'imports unused en haut
01:30:23 <seirl> et c'est toi qui me casse les couilles quand j'oublie d'avoir syntastic sur la machine que j'utilise
01:30:59 <seirl> donc euh zopieux je suppose que je dois écrire "wb" à la place de "rb" ?
01:31:23 <seirl> non mais attends je comprends juste pas
01:31:34 <zopieux> Mec
01:31:36 <zopieux> C'était juste
01:31:36 <seirl> ok faut aussi commenter la ligne d'en dessous
01:31:41 <seirl> compris
01:31:48 <zopieux> Pour pas avoir à télécharger à chaque lancement
01:31:49 <delroth> "rb" je vais prononcer ça "reubeu" à partir de maintenant
01:31:52 <seirl>     for key, value in emoji_db.items()
01:31:52 <seirl> KeyError: 'unicode'   
01:31:56 <zopieux> Un cache de debug quoi
01:32:01 <seirl> they see me just decommenting
01:32:03 <seirl> they hating
01:32:32 <zopieux> Mais commité car je suis très incompétent
01:32:48 <seirl> https://raw.githubusercontent.com/Ranks/emojione/master/emoji.json
01:32:55 <seirl> y'a pas de field "unicode"
01:32:56 <seirl> dans tout le json
01:32:57 <zopieux> Ils ont changé le format ?
01:33:06 <seirl> juste unicode_version   
01:33:22 <zopieux> Bah bienvenue dans le monde merveilleux de la communauté js
01:33:32 <seirl> c'est "ascii" maintenant j'imagine ?
01:33:39 <zopieux> Pour leur défense on est pas censé utiliser leur merde comme ça                                                                                                                                                 
01:34:14 <zopieux> seirl: shortnamz non
01:34:25 <zopieux> Je l'utilise pour quoi ce Unicode
01:34:38 <zopieux> Faut juste le bon nom de fichier et le bon nom entre : :                                                                                                                                                        
01:35:00 <seirl> f.write(repr(emoji_db)) 
01:35:10 <zopieux> Quoi   
01:35:26 <seirl> mec      
01:35:31 <seirl> comment je bypass cette merde 
01:35:35 <seirl> j'ai autre chose à foutrer
01:36:48 <seirl> on peut pas faire un truc genre si y'a pas les emoji ça affiche un fucking warning et ça continue                                                                                                                 
01:38:26 <seirl> /home/antoine/prologin/site/assets/libs.sh: line 49: -p: command not found                                                                                                                                        
01:38:35 <seirl> jvais tellement te péter la gueule
01:39:00 <seirl> NODEJS=$(which node || which nodejs)
01:39:06 <seirl> $NODEJS -p
01:39:21 <seirl> c'est bien pas quoté comme il faut 
01:39:25 <seirl> histoire de maximiser l'échec
01:41:20 <seirl> va savoir pourquoi on a besoin de nodejs pour générer des statics aussi

```
